eng/pipelines/extra-platforms/runtime-extra-platforms-wasm.yml

```
@@ -213,6 +213,17 @@ jobs:
       # ff tests are unstable currently
       shouldContinueOnError: true
 
+  - template: /eng/pipelines/common/templates/wasm-debugger-tests.yml
+    parameters:
+      platforms:
+        - Browser_wasm
+        - Browser_wasm_win
+      extraBuildArgs: /p:MonoWasmBuildVariant=multithread /p:WasmEnableThreads=true
+      nameSuffix: DebuggerTests_MultiThreaded
+      alwaysRun: true # always run when invoked manually
```

```suggestion
      alwaysRun: ${{ parameters.isWasmOnlyBuild }}
```

This will run whenever the pipeline is invoked manually.